rpc_jobs/rpc_aio.yml

```diff
@@ -300,6 +300,7 @@
             if(common.is_doc_update_pr("${{env.WORKSPACE}}/rpc-openstack")){{
               return
             }}
+            maas.add_maas_env_vars()
```

Comment:
Can you add a comment above this line to express what this is doing. It just makes the logic easier to follow later.